Lane Changes/ Vehicle Settings (#289)
monoDriveIO/documentation: docs/scenario_editor/scenarios.md

- **Steer Forward Speed:** The range of speed (mph) used to LERP `steerForwardDistance`.

- **Rolling stop speed:** The speed to travel before continuing after a legal stop.

- **Distance to Stop Sign:**  The number of centimeters to stop before a controlled intersection.

- **Spawn Offset:** The offset to apply to the vehicle position prior to spawning.

- **Deceleration Constraints:** The minimum and maximum deceleration of the vehicle used by AutoPilot. cm/s^2.

- **Turn Pref:** (Turn Preference) At the next turn the vehicle will prefer to take this turn. 
	This will reset the drop down menu to "NONE" after the following turn.

- **Simple Speed Look Ahead:** The number of centimeters to look ahead when performing simple speed control.

- **Vehicle Length:** Total length of the vehicle, used in various maneuver calculations.

- **Point PID:** PID Controller settings for point control.

- **Speed PID:** PID Controller settings for speed control.

- **Lane Offset:**  The number of centimeters to drive from the lane center: positive is right, negative is left.

- **Changing Lanes Left: / Changing Lanes Right:**  Status change only, shows when vehicle is in lane change.

> -	**Lane Change Transition Distance:** Lane Change is considered complete after the vehicle coordinate frame is close to the new lane center.

Reviewer: ```suggestion
-	**Lane Change Transition Distance:** Lane Change is considered complete after the origin vehicle coordinate frame is this many centimeters from the new lane center.
```

> -	**Steer Forward Distance During Lane Change:** The range of centimeters to look ahead when performing steering AI actions during a lane change, interpolated based on steerForwardSpeedRange.

Reviewer: ```suggestion
-	**Steer Forward Distance During Lane Change:** The range in centimeters to look ahead when performing steering AI actions during a lane change, interpolated based on steerForwardSpeedRange.
```

> - **Clamp Decay Exponent:** Exponential decay was added to lighten the clamping to 1 as the vehicle converged on the center lane to regain full steering control.

Reviewer: ```suggestion
- **Clamp Decay Exponent:** Exponential decay to lighten the clamping to full steering extents as the vehicle converges on the center lane to regain steering control.
```

- **Ignores Lane Change Boxes:**  If set, all lane change trigger boxes will be ignored.

- **Ignore Other Vehicles:**  If set, all other vehicles will be ignored when computing AI actions.

- **Ignore Traffic Signals:**  If set, this vehicle will ignore all traffic control devices.

- **Obey Road Speed Limits:** If set, this vehicle will obey the speed limits of the road and
set speed and initial speed will be a delta above or below the road's speed limit. 

- **Limit Speed in Curves:**  If set, this vehicle will attempt to slow down in order to properly corner in turns.

- **Initial Speed is Delta:** If set, when vehicle is obeying road speed limits, the initial velocity will be used as a delta above or below the speed limit. If vehicle is not obeying road speed limits this setting is ignored.

- **Destroy on Collision:**  If set, this vehicle will be removed in the scene after colliding with another physics body.

- **Should Overtake Slower Vehicles:** If set, will allow vehicle to overtake slower vehicles that it is following for the overtake timer trigger set amount of time. Otherwise, the vehicles will slow to the lead vehicle's speed and remain there. 

> - **Overtake Timer Trigger:** The time in seconds a vehicle should wait before triggering the should overtake slower vehicles behavior when it is following a vehicle. Only applies when the vehicle is allowed to overtake vehicles.

Reviewer: ```suggestion
- **Overtake Timer Trigger:** The time in seconds a vehicle should wait before triggering the "Should Overtake Slower Vehicles" behavior when it is following a vehicle. Only applies when the vehicle is allowed to overtake vehicles.
```
